src/styles/catalog.scss

```diff
@@ -0,0 +1,156 @@
+.catalog__product,
+.catalog {
+  display: flex;
+}
+
+.catalog__model,
+.catalog__code,
+.catalog__count,
+.catalog__name,
+.price__value {
+  margin: 0;
+}
+
+.catalog {
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 944px;
+  margin: 0 auto;
+  padding-top: 50px;
+
+  &__product {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: #fff;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    width: 166px;
+    align-items: center;
+    padding: 32px 16px 16px;
+    flex-wrap: wrap;
+    margin-bottom: 46px;
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__model {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    color: #060b35;
+
+    cursor: default;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    padding-bottom: 4px;
+  }
+
+  &__code {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__review {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
```

Comment:
you don`t need to write font-family in each class, you just need to write in once in html